notifications/09-08-20-0900.md

```diff
@@ -0,0 +1,27 @@
+
+**What is happening?**
+
+MySC CHG0017053
```

Comment:
```suggestion
MySC CHG0030700
```

The original Change reference is for internal DXCAS ticketing system. When referring to MySC you need to look at the case exchange info in the RFC, which is what BC Gov stakeholders would be able to look at for this RFC.